api/environments/environment_controller_test.go

```diff
@@ -274,6 +274,16 @@ func TestGetEnvironmentSummary_OrphanedEnvironmentWithDash_OrphanedEnvironmentIs
 		WithEnvironment("dev", "master").
 		WithEnvironment(anyOrphanedEnvironment, "feature"))
 
+	commonTestUtils.ApplyEnvironment(builders.
```

Comment:
Would have deleted the line:
WithEnvironment(anyOrphanedEnvironment, "feature")

... and this ApplyEnviornment